spec/depth-enabled-photography.html

```diff
@@ -2160,121 +2160,417 @@
+            photoUtils.photoCrop(photo, { x: originX, y: originY, w: width, h: height}).then(
+                function(croppedPhoto) {
+                  croppedPhoto.queryContainerImage().then(
+                      function(image) {
+                        // Render the cropped photo image.
+                      },
+                      function(error) { console.log(error.message); });
+                },
+                function(error) { console.log(error.message); });
+          }
+
+          function photoRotate() {
+            // Rotate photo by 90 degree clockwise.
+            photoUtils.photoRotate(photo, 90.0).then(
+                function(rotatedPhoto) {
+                  rotatedPhoto.queryContainerImage().then(
+                      function(image) {
+                        // Render the rotated photo image.
+                      },
+                      function(error) { console.log(error.message); });
+                },
+                function(error) { console.log(error.message); });
+          }
+        </pre>
+      </section>
+      <section>
+        <h3>
+          More Photo Process
+        </h3>
+        <pre class='example highlight'>
+          // photo is the instance of DepthPhoto
+          // which can be got from the camera [Example 1] or loaded from somewhere [Example 2 or 3]
+
+          // Measurement
+          var measurement;
+          try {
+            measurement = new realsense.DepthEnabledPhotography.Measurement();
+          } catch (error) {
+            console.log(error.message);
+            return;
+          }
+          // startX, startY, endX and endY defines the start and end point of the measurement respectively.
+          measurement.measureDistance(photo, { x: startX, y: startY }, { x: endX, y: endY }).then(
+              function(d) {
+                console.log(
+                    'Distance between(' + startX + ',' + startY +
+                    ') - (' + endX + ',' + endY + ') = ' +
+                    parseFloat(d.distance).toFixed(2) + ' millimeters, Confidence = ' +
+                    parseFloat(d.confidence).toFixed(2) + ', Precision=' +
+                    parseFloat(d.precision).toFixed(2) + 'mm');
+              },
+              function(error) { console.log(error.message); });
+
+
+          // Motion Effect
+          var motionEffect;
+          try {
+            motionEffect = new realsense.DepthEnabledPhotography.MotionEffect();
+          } catch (error) {
+            console.log(error.message);
+            return;
+          }
+          motionEffect.init(photo).then(
+              function() {
+                doMothionEffect();
+              },
+              function(error) { console.log(error.message); });
+
+          function doMothionEffect() {
+            var right = 0.0, up = 0.0, forward = 0.0;
+            var yaw = 0.0, pitch = 0.0, roll = 0.0;
+            var zoom = 0.0;  // Set these motion parameters at your convenience.
```

Comment:
How about give an example set of parameters to demonstrate a use case? e.g. change the view point some degree by setting pitch?